### PR TITLE
Set default tee device name to NULL

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Depends on https://github.com/OP-TEE/optee_client/pull/47

### CODE REVIEW

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: host/xtest/xtest_main.c [hunk 1]
```diff
@@ -114,7 +114,7 @@ ADBG_SUITE_ENTRY(XTEST_TEE_BENCHMARK_1003, NULL)
 ADBG_SUITE_DEFINE_END()
 
 static const char gdevname_tz[] = "opteearmtz00";
-char *_device = (char *)gdevname_tz;
+char *_device = NULL;
```
Comment (pascal-brand-st-dev) [inline host/xtest/xtest_main.c]: `gdevname_tz` can be removed, and the further dump of default value updated

**Comment by jenswi-linaro** [comment]
Update

**Comment by pascal-brand-st-dev** [comment]
Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

**Comment by jenswi-linaro** [comment]
Rebased, tags applied

**Pull request merged**